src/graphql/resolvers.js

```diff
@@ -1,17 +1,44 @@
 import {
   getContributorInfo,
+  getOpenIssuesByLabel,
   isGitHubContributor,
   inviteIfNecessary
 } from '../lib/github';
-import { createShopifyCustomer } from '../lib/shopify';
+import { createShopifyCustomer as createShopifyCustomerRest } from '../lib/shopify';
+import { createShopifyCustomer } from '../lib/shopify-graphql';
 import getLogger from '../lib/logger';
+import { prisma } from '../prisma-client';
 
 const logger = getLogger('graphql/resolvers');
 
 export default {
   Query: {
     contributorInformation: async (_, { githubUsername }) => {
       return await getContributorInfo(githubUsername);
+    },
+    openIssues: async (_, { label }) => {
+      return await getOpenIssuesByLabel(label);
+    },
+    getContributor: async (_, { githubUsername }) => {
+      // TODO when should we create the contributor the first time?
+      // const resp = await prisma.createContributor({
+      //   githubUsername: 'jlengstorf',
+      //   email: 'jason@gatsbyjs.com'
+      // });
+
+      const [contributor, contributorInfo] = await Promise.all([
+        prisma.contributor({ githubUsername }),
+        getContributorInfo(githubUsername)
+      ]);
+
+      if (!contributor || !contributor.githubUsername) {
```

Comment:
So, I'm thinking: 

1. if there's no contributor record, we should return the github info and github username
  - this would require making `email` nullable, which is fine -- we treat it like the `shopifyCustomerId`. If they haven't submitted the form, and there's no user record, all we have is their github sign in. Makes sense.
2. If there is a record, return the contributor and github info.

Reply:
Yeah, that makes sense. We get the email from GitHub, but it makes sense to let contributors choose which email they want to set up in Shopify and get the newsletter at (if they opt in).

Reply:
Exactly